investpy/currency_crosses.py

```diff
@@ -0,0 +1,248 @@
+        IOError: raised if the currencies file from `investpy` is missing or errored.
+    """
+
+    resource_package = __name__
+    resource_path = '/'.join(('resources', 'currency_crosses', 'currency_crosses.csv'))
+    if pkg_resources.resource_exists(resource_package, resource_path):
+        currencies = pd.read_csv(pkg_resources.resource_filename(resource_package, resource_path))
+    else:
+        currencies = retrieve_currency_crosses()
+
+    if currencies is None:
+        raise IOError("ERR#0037: currencies not found or unable to retrieve.")
+
+    return currencies['name'].tolist()
+
+
+def currency_crosses_as_dict(columns=None, as_json=False):
+    """
+    This function retrieves all the available currencies on Investing.com and returns them as a :obj:`dict` containing the
+    `country`, `name`, `full_name`, `symbol`, `tag` and `currency`. All the available currencies can be found at:
+    https://es.investing.com/currencies/
+
+    Args:
+        country (:obj:`str`, optional): name of the country to retrieve all its available currencies from.
+        columns (:obj:`list` of :obj:`str`, optional): description
+            a :obj:`list` containing the column names from which the data is going to be retrieved.
+        as_json (:obj:`bool`, optional): description
+            value to determine the format of the output data (:obj:`dict` or :obj:`json`).
+
+    Returns:
+        :obj:`dict` or :obj:`json` - currencies_dict:
+            The resulting :obj:`dict` contains the retrieved data if found, if not, the corresponding
+            fields are filled with `None` values.
+
+            In case the information was successfully retrieved, the :obj:`dict` will look like::
+
+                {
+                    'country': country,
+                    'name': name,
+                    'full_name': full_name,
+                    'symbol': symbol,
+                    'tag': tag
+                }
+
+    Raises:
+        ValueError: raised when the introduced arguments are not correct.
+        IOError: raised if the currencies file from `investpy` is missing or errored.
+    """
+
+    if not isinstance(as_json, bool):
+        raise ValueError("ERR#0002: as_json argument can just be True or False, bool type.")
+
+    resource_package = __name__
+    resource_path = '/'.join(('resources', 'currency_crosses', 'currency_crosses.csv'))
+    if pkg_resources.resource_exists(resource_package, resource_path):
+        currencies = pd.read_csv(pkg_resources.resource_filename(resource_package, resource_path))
+    else:
+        currencies = retrieve_currency_crosses()
+
+    if currencies is None:
+        raise IOError("ERR#0037: currencies not found or unable to retrieve.")
+
+    if columns is None:
+        columns = currencies.columns.tolist()
+    else:
+        if not isinstance(columns, list):
+            raise ValueError("ERR#0020: specified columns argument is not a list, it can just be list type.")
```

Comment:
Error codes cannot be duplicated, so the error code should be formatted as `ERR#xxxx` and the code should be a different one for each kind of error.